src/localstack/mod.rs

```diff
@@ -1,12 +1,16 @@
+pub use pro::LocalStackPro;
 use testcontainers::{core::WaitFor, Image};
 
+/// LocalStack Pro
+pub mod pro;
+
 const NAME: &str = "localstack/localstack";
-const TAG: &str = "3.0";
+const TAG: &str = "3.8";
```

Comment:
We usually consider version bumps as a breaking change, because unexpected regressions may happen even for minor updates

If it's possible to just update version without changes from our side - user always can do that by specifying particular version and that's even recommended approach

Reply:
Just to get it on the record: there's an argument to be made that the pinned image versions should be kept *reasonably* up to date, (maybe on some kind of standard cadence that'd reflected in the crate version? 🤔), and I'd absolutely be one to make that argument.

You're right though, about it being a breaking change. Since this PR is intended to be a patch version bump at most, I just went ahead and reverted the version bump and updated the PR description 🙂.